### PR TITLE
Hello-Microverse-project pull request

### PR DESCRIPTION
**Pull request by suthish777**
This is the Pull request for the Hello-Microverse project in this project I added all the necessary files and also added the MIT license and passed all the checks and this branch has no conflicts with the base branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,285 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (AlphaNtihinduka) [inline README.md:283]: - [ ] I have noticed that the MIT link is broken. The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. Kindly ensure that the MIT link is properly working and leading to the proper license ☺️ .

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @suthish777 🙋‍♂️
Good job working on your first contact form. You have done the design to the best expectation and incredibly well. 👏  

However, there are some issues mostly on MIT and pull request description.

You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

# Required changes ♻️ 
- [ ] You have so far done great by having a pull request description ☺️  though not yet descriptive. Kindly ensure that the Pull Request have a summary describing introduced changes in more detail 👍 .

## Optional suggestions
- [ ] [OPTIONAL] Please ensure that the feature branch is more descriptive. I would suggest you consider the use of `hello_microverse` to five a little overview of what the branch is all about.
 ☺️ 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: README.md (hunk 1 above)
Comment (suthish777) [reply README.md:283]: Can you help how to fix this issue, I added a LICENSE.md file is that okay.

Comment (suthish777) [reply README.md]: I fixed the broken MIT link as you suggested.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 ## Hi @suthish777 ,
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is now okay :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 * Good job :+1: 
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:


 
 ## Optional suggestions

- [ ] N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**